recipes/gseapy/meta.yaml

```diff
@@ -1,45 +1,44 @@
 package:
   name: gseapy
-  version: "0.12.0"
+  version: "0.12.1"
 
 source:
-  url: https://files.pythonhosted.org/packages/6f/3c/6e06d23461a49554d179c885614e2f59706e9176cbc328deda0836952d71/gseapy-0.12.0.tar.gz
-  sha256: 6f9ea045ddc40068650db8c4f5668010004787dca9bc8c5ed1757cb8313b4765
+  url: https://files.pythonhosted.org/packages/d2/e3/ea5f9c0f4f6a278de7ebc80fba9e378fb70b037a3766ea4cea9b16a28203/gseapy-0.12.1.tar.gz
+  sha256: 121d973168a54143da0d1ecdfeb70c34e631af90f58f602731a1db08206375e0
 
 build:
-  noarch: python
   entry_points:
     - gseapy = gseapy.__main__:main
+  skip: True  # [py2k or py == 36 ]
   number: 0
 
 requirements:
+  build: 
+    - {{ compiler('rust') }}
   host:
-    - rust
-    - python >3
+    - python
+    - wheel
     - setuptools
     - setuptools-rust
     - numpy >=1.13.0
```

Comment:
The `host:` section should usually only list dependencies required to run `pip install`, so you can most likely omit `scipy`, `pandas` etc. which are only needed at runtime.

Reply:
There were some strange CI errors, so I took the liberty of pushing these changes also to re-trigger CI.

Reply:
Any thougths to make MacOS build work ? I managered to build it on my MacOS with bioconda-utils.

Reply:
Sorry, I have no idea. The log output is unfortunately very unhelpful. Let’s focus on #36469 instead.